karamel-core/src/test/resources/cookbooks/karamelchef/kagent-chef/master/recipes/default.rb

```diff
@@ -0,0 +1,1135 @@
+
```

Comment:
Nice spot. This is a testing class. I was downloading recipes using wget from github raw urls. Obviously something went wrong. The test still passes though, because all it needs is a recipe file to parse....